### ISSUE TITLE
what's the difference between val and test model?

### ISSUE THREAD

**Issue by ForestWang** [XuJiacong/PIDNet#16]

**Comment by XuJiacong** [comment]
The val model was trained by training set only, but the test model was trained by training and validation set together.
